### PR TITLE
added Racket online tools

### PR DESCRIPTION
**Pull request by spdegabrielle**
Added rollApp version of DrRacket IDE to 'Full IDE'
Added pasterack racket-evaluating pastebin to 'Snippets' 
Added period to last line of `Appitr` IDE entry in line with Contribution Guidelines

### CODE REVIEW

**Comment by spdegabrielle** [comment]
removed trailing slash where reported by travis-ci

**Comment by styfle** [comment]
The description does not comply with number 7 in [contributing.md](https://github.com/styfle/awesome-online-ide/blob/master/contributing.md)

7. Add a short description for the IDE
- Begin with the word "Run" followed by the primary language or technology, (i.e. Run Racket...)
- Explain how this IDE is different/better than the competition

**Comment by spdegabrielle** [comment]
Updated as requested.

File: README.md [hunk 1]
```diff
@@ -48,7 +48,8 @@ An ([IDE](https://en.wikipedia.org/wiki/Integrated_development_environment)) is
...
-- [Appitr](https://appitr.com) <img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run JavaScript ES6 with React Native in the browser built on [Monaco Editor](https://microsoft.github.io/monaco-editor/) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/Microsoft/monaco-editor) and React Native Web
+- [Appitr](https://appitr.com) <img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run JavaScript ES6 with React Native in the browser built on [Monaco Editor](https://microsoft.github.io/monaco-editor/) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/Microsoft/monaco-editor) and React Native Web.
+- [DrRacket](https://www.rollapp.com/app/drracket) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run [Racket](https://racket-lang.org) programming language. DrRacket includes macro debugging not present in any other IDE.
```
Comment (styfle) [inline README.md]:
```suggestion
- [DrRacket](https://www.rollapp.com/app/drracket) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run [Racket](https://racket-lang.org) with support for macro debugging.
```

**Pull request merged**